### ISSUE TITLE
Docker Hub Webhook Deployment

### ISSUE THREAD

**Issue by Rdonn** [caprover/caprover#1324]
Dockerhub provides webhooks. This application already has integration with SCM provider webhooks and has an option to deploy directly off of an image; I think a being able to respond to remote repositories would be a very valuable feature.

**Comment by marekbrze** [comment]
Yes, I have the same problem. I use Nx monorepo and it would be the easiest solution to deploy apps.

**Comment by newsaeta** [comment]
I have the same opinion

**Comment by yixiongngvsys** [comment]
i have the same opinion, I need it for auto deployment whenever a newer images is build from docker hub
